projects/packages/analyzer/src/api/class-analyze-controller.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace Automattic\Jetpack\Analyzer;
+
+class Analyze_Controller extends Controller {
+	public function get() {
+		try {
+			Locker::lock();
+		} catch ( \Throwable $th ) {
+			// TODO: extend default Exception class, and use instance_of instead.
+			if ( 'Locked already' === $th->getMessage() ) {
+				return $this->model->get_status();
+			}
+		}
+
+		$cmd = 'php ' . dirname( __DIR__ ) . '/../scripts/jp-analyze-parallel.php';
```

Comment:
Quoting again.

Also you may need to take some measures to avoid having the process be killed when the web request exits. Maybe making the command be like `nohup php /path/to/scripts/jp-analyze-parallel.php &` instead would be sufficient.

Reply:
When the webserver process finishes, it may send SIGHUP to all the child processes which may cause them to exit. `nohup` avoids that.